policyengine_us/variables/gov/states/oh/tax/income/credits/lump_sum_distribution/oh_distribution_credit.py

```diff
@@ -0,0 +1,22 @@
+from policyengine_us.model_api import *
+
+
+class oh_distribution_credit(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Ohio lump sum distribution credit"
+    unit = USD
+    definition_period = YEAR
+    reference = (
+        "https://tax.ohio.gov/static/forms/ohio_individual/individual/2021/pit-it1040-booklet.pdf#page=29",
+    )
+    defined_for = StateCode.OH
+
+    def formula(tax_unit, period, parameters):
+        person = tax_unit.members
+        p = parameters(period).gov.states.oh.tax.income.credits
+        age = person("age", period)
+        amount = p.lump_sum_distribution.amount
+        age_threshold = p.senior_citizen.age_threshold
+        rate = p.lump_sum_distribution.rate.calc(age)
+        return where(age >= age_threshold, amount * rate, 0)
```

Comment:
```suggestion
        return where(age >= age_threshold, p.amount * rate, 0)
```

Reply:
The logic of the variable seems incomplete when compared with the worksheet in this link: https://tax.ohio.gov/static/forms/ohio_individual/individual/2021/pit-it1040-booklet.pdf#page=29
![image](https://github.com/PolicyEngine/policyengine-us/assets/89811054/8b21bfbd-72b3-41c1-b5ac-516cc2dbe54a)
and I think we at least need to create a breakdown parameter to represent the following Table 2:
![image](https://github.com/PolicyEngine/policyengine-us/assets/89811054/617d2ca5-33b9-4a68-b5d9-e8d610e44f23)